helm/cluster-cloud-director/templates/kubeadmconfigtemplate.yaml

```diff
@@ -1,15 +1,17 @@
+{{- range .Values.nodePools }}
+---
 apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
 kind: KubeadmConfigTemplate
 metadata:
-  name: {{ include "resource.default.name" $ }}-md0
-  namespace: {{ .Release.Namespace }}
+  name: {{ include "resource.default.name" $ }}-{{ .name }}
```

Comment:
Please check https://github.com/giantswarm/giantswarm/issues/22246

Reply:
You're right, I might have been using an older commit for some reason, thanks

Reply:
Fixed in https://github.com/giantswarm/cluster-cloud-director/pull/9/commits/bd23a7a32bcdb8e42e21e6968ec8e90babe0aa52 can you review @erkanerol ?